packages/grpc-js/src/channelz.ts

```diff
@@ -348,105 +330,102 @@ export interface SocketInfo {
   remoteFlowControlWindow: number | null;
 }
 
-interface ChannelEntry {
+type ChannelEntry = {
```

Comment:
Why replace these interfaces with types?

Reply:
that was actually from another attempt to use a doubly linked list, I'll revert this